pkg/tnf/handlers/liveness/liveness.go

```diff
@@ -0,0 +1,57 @@
+package liveness
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/test-network-function/test-network-function/internal/ocpclient"
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Liveness struct {
+	timeout       time.Duration
+	podName       string
+	containerName string
+	namespace     string
+	liveness      bool // true if container has liveness section defined
+	Result        int
+}
+
+func NewLiveness(ns, pod, container string, timeout time.Duration) Liveness {
+	return Liveness{
+		timeout:       timeout,
+		podName:       pod,
+		containerName: container,
+		namespace:     ns,
+		liveness:      false,
+		Result:        tnf.ERROR,
+	}
+}
+
+func (l *Liveness) RunTest() {
+	cl := ocpclient.GetOcpClient()
```

Comment:
Instead of getting a new OcpClient here, from my other comment just maybe use the client passed along in the Liveness struct?

Reply:
hmm I did not get the point. I'm not getting a new OcpClient. I'm getting "THE" OcpClient